salt/etcd-proxy/etcd-proxy.conf.jinja

```diff
@@ -4,7 +4,7 @@
 ETCD_DATA_DIR="/var/lib/etcd/"
 ETCD_NAME="{{ grains['id'] }}"
 
-ETCD_LISTEN_CLIENT_URLS="http://0.0.0.0:2379"
+ETCD_LISTEN_CLIENT_URLS="http://127.0.0.1:2379"
 ETCD_LISTEN_PEER_URLS="https://0.0.0.0:2380"
 ETCD_ADVERTISE_CLIENT_URLS="http://{{ grains['caasp_fqdn'] }}:2379"
```

Comment:
@inercia : I would agree with you, but I think that `grains['caasp_fqdn']` does not point on `127.0.0.1`. So with this setup we will get situation when etcd cluster is advertising wrong client URL